test/01_start_vm.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Setting up bridge for VMs"
+ip link add name vm-br0 type bridge || true
+ip link set up dev vm-br0 || true
+ip addr add 100.100.0.1/24 dev vm-br0 || true
+
+echo "Setting up tap device for VM"
+ip tuntap add mode tap name tap0 || true
+ip link set tap0 up || true
+ip link set tap0 master vm-br0 || true
+
+# kernels shipped with ubuntu based images allow for direct kernel boot without passing initrd to cloud-hypervisor
+if [[ "${OS_NAME}" == "ubuntu" ]]; then
+  INITRAMFS=""
+  KERNEL="os-kernel"
+elif [[ "${OS_NAME}" == *"firewall" ]]; then
+  INITRAMFS=""
+  KERNEL="os-kernel"
+elif [[ "${OS_NAME}" == "debian" || "${OS_NAME}" == "debian-nvidia" ]]; then
+  INITRAMFS=""
+  KERNEL="metal-kernel"
+else
+  INITRAMFS="--initramfs ./initramfs"
+  KERNEL="os-kernel"
+fi
```

Comment:
Almalinux is missing here ?

Reply:
It's the safest config that should work for most operating systems.
Therefore I did not mention almalinux directly.